### PR TITLE
Update to 1.19.4

### PR DESCRIPTION
**Pull request by woodiertexas**

### CODE REVIEW

**Comment by xanderstuff** [comment]
Perhaps these commits should be squashed into 1 commit called "Update to 1.19.4" before they are merged to main? (to have a cleaner/more organized commit history that we can reference in case there are regressions, ect.)

**Comment by xanderstuff** [comment]
~~And I suppose it would be nice if the auto-format stuff would be in a separate, dedicated commit, but oh well.~~

Edit: oh, actually I guess that's all in "fixed formatting issues" 0cec5366997fb44633cd6b25517386eff3f8de73. In that case, maybe squash all 1.19.4-related commits together (into "Update to 1.19.4") except for that autoformat commit? ie just turn this into 2 commits, if that's possible. Or 3 commits if you want the version bump to be separate as well

File: src/main/java/io/github/debuggyteam/tablesaw/TableSawRecipes.java [hunk 1]
```diff
@@ -1,24 +1,19 @@
...
 import com.google.common.collect.Multimap;
-
 import io.github.debuggyteam.tablesaw.api.TableSawRecipe;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemConvertible;
 import net.minecraft.item.ItemStack;
 
+import java.util.*;
```
Comment (Ampflower) [inline src/main/java/io/github/debuggyteam/tablesaw/TableSawRecipes.java]: Star import intended?

File: src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java [hunk 2]
```diff
@@ -1,24 +1,7 @@
...
-import org.jetbrains.annotations.NotNull;
-import org.quiltmc.loader.api.QuiltLoader;
-import org.quiltmc.qsl.resource.loader.api.reloader.SimpleSynchronousResourceReloader;
-
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-import com.google.gson.JsonPrimitive;
+import com.google.gson.*;
```
Comment (Ampflower) [inline src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java]: Star import intended?

File: src/main/java/io/github/debuggyteam/tablesaw/TableSaw.java [hunk 3]
```diff
@@ -23,58 +24,57 @@
...
+		LOGGER.info("Hello Quilt world from {}!", mod.metadata().name());
+		
+		Registry.register(Registries.SCREEN_HANDLER_TYPE, new Identifier(MODID, "tablesaw"), TABLESAW_SCREEN_HANDLER);
+		
+		Registry.register(Registries.BLOCK, new Identifier(MODID, "tablesaw"), TABLESAW);
+		Registry.register(Registries.ITEM, new Identifier(MODID, "tablesaw"), 
+				new BlockItem(TABLESAW, new QuiltItemSettings()));
+		ItemGroupEvents.modifyEntriesEvent(ItemGroups.FUNCTIONAL_BLOCKS).register(entries -> entries.addItem(TABLESAW));
+		
+		//Registry.register(Registries.SOUND_EVENT, TableSaw.TABLESAW_SFX, TABLESAW_SOUND_EVENT);
```
Comment (Ampflower) [inline src/main/java/io/github/debuggyteam/tablesaw/TableSaw.java]: Why is this commented out?

**Review by Ampflower — COMMENTED** [review]
Noticed a couple things that seemed out of place; particularly using star imports (intentional?) and not registering the sound event.

File: src/main/java/io/github/debuggyteam/tablesaw/TableSaw.java (hunk 3 above)
Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/tablesaw/TableSaw.java]: I wasn't sure of the 1.19.4 equivalent for sound events.

File: src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java (hunk 2 above)
Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java]: Probably not, I think intelliJ did it when I used ctrl+alt+o to optimize imports.

File: src/main/java/io/github/debuggyteam/tablesaw/TableSawRecipes.java (hunk 1 above)
Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/tablesaw/TableSawRecipes.java]: Likely happened due to import optimization.

File: src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java (hunk 2 above)
Comment (xanderstuff) [reply src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java]: That's probably intellij's default autoformat settings, which change 3+ similar imports to that (or was it 4+ ...?)

Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java]: 3-5 imports get turned into star imports by default. My solution to that was this: 
image

File: src/main/java/io/github/debuggyteam/tablesaw/TableSaw.java (hunk 3 above)
Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/tablesaw/TableSaw.java]: Resolved with https://github.com/DebuggyTeam/tablesaw/pull/22/commits/510ff42f5d04bc2aa75bb94980d1071633385a5e

**Review by Ampflower — APPROVED** [review]
Looks good to me

**Pull request merged**